src/Agent.Worker/StepsRunner.cs

```diff
@@ -33,6 +34,19 @@ public interface IStepsRunner : IAgentService
 
     public sealed class StepsRunner : AgentService, IStepsRunner
     {
+        private IJobServerQueue _jobServerQueue;
+
+        private IJobServerQueue JobServerQueue
```

Comment:
It can be simplified
`private IJobServerQueue JobServerQueue => _jobServerQueue ??= HostContext.GetService<IJobServerQueue>();`